spd/losses.py

```diff
@@ -22,12 +28,29 @@
     ci_masked_recon_subset_loss,
     faithfulness_loss,
     importance_minimality_loss,
+    pgd_recon_layerwise_loss,
+    pgd_recon_loss,
+    pgd_recon_subset_loss,
     stochastic_hidden_acts_recon_loss,
     stochastic_recon_layerwise_loss,
     stochastic_recon_loss,
     stochastic_recon_subset_loss,
 )
 from spd.models.component_model import CIOutputs, ComponentModel
+from spd.scheduling import get_cosine_schedule_value, get_linear_schedule_value
+
+
+def get_loss_coeff(
```

Comment:
maybe just call this get_coeff, as we might use it for things that aren't loss coefficients in the future.